spec/bundler/endpoint_specification_spec.rb

```diff
@@ -62,4 +62,8 @@
       end
     end
   end
+
+  it "supports equality comparison" do
+    expect(subject.eql?(subject)).to eq(true)
```

Comment:
you can use `expect(subject).to eel(subject)` here, although I think it would be helpful to test `eql` against a different instance?